helm/oauth2-proxy/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: oauth2-proxy
-version: 7.7.30
+version: 7.7.31
 apiVersion: v2
-appVersion: 7.7.1
+appVersion: 7.7.2
```

Comment:
Please revert. There is no v7.7.2 for oauth2-proxy as of yet

Reply:
Done.